### PR TITLE
Hotfix292 close nc

### PR DESCRIPTION
**Pull request by akorosov**
@korvinos , can you please check one more PR? This is related to the same issue #292 that some tests do not pass on Windows when conda recipe is built.
I've added continuous integration with AppVeyor that builds nansat on windows. Finally all tests pass! That means that on conda they will also pass.
I also added AppVeyor badge to README. The badge shows the master branch status. It has status 'unknown' yet, but should be green after we merge with master.
As usual, please don't delay too much. Thank you!

### CODE REVIEW

**Comment by akorosov** [comment]
By the way, there is an easy way to compare overview changes (and not commit by commit):
https://github.com/nansencenter/nansat/compare/hotfix292-close-nc

File: nansat/tests/test_exporter.py [hunk 1]
```diff
@@ -15,11 +15,13 @@
 import os
 import sys
 import json
+import time
 import logging
 import unittest
 import warnings
 import datetime
 import tempfile
+from pprint import pprint
```
Comment (korvinos) [inline nansat/tests/test_exporter.py]: Is that so necessary now? I think, if you find it's good then you should do it generally in development and make it a standard for all print funcs.

File: nansat/tests/test_exporter.py [hunk 2]
```diff
@@ -332,18 +328,15 @@ def test_export_netcdf_complex_remove_meta(self):
...
     def test_export_netcdf_arctic(self):
-        ''' Test export of the arctic data without GCPS
-        '''
         n = Nansat(self.test_file_arctic)
         n.export(self.tmpfilename)
         exported = Nansat(self.tmpfilename)
         self.assertTrue((n[1] == exported[1]).any())
         self.assertTrue((n[2] == exported[2]).any())
         self.assertTrue((n[3] == exported[3]).any())
+        time.sleep(1)
```
Comment (korvinos) [inline nansat/tests/test_exporter.py]: What is a reason for that?

File: nansat/tests/test_exporter.py [hunk 3]
```diff
@@ -264,28 +263,27 @@ def test_export2thredds_arctic_long_lat(self):
...
-        self.assertTrue(float(southernmost_latitude) > 53,
-                        'southernmost_latitude is wrong:' +
-                        southernmost_latitude)
+        pprint(metadata)
+
+        # Test that the long/lat values are set correctly
+		# GDAL behaves differently:
+		# Windows: nc-attributes are accessible without 'NC_GLOBAL#' prefix
+		# Linux: nc-attributes are accessible only with 'NC_GLOBAL#' prefix
+        if 'linux' in sys.platform:
```
Comment (korvinos) [inline nansat/tests/test_exporter.py]: Are this changes necessary only for the passing of the test, and if so,  will this work ok with windows in real life? Could you educate me a little bit or point to the code, please, because I do not clearly understand that?

File: nansat/tests/test_exporter.py (hunk 2 above)
Comment (aleksandervines) [reply nansat/tests/test_exporter.py]: unit tests should ideally take miliseconds. Adding a 1 second sleep like this seems to really break that :)

File: nansat/tests/test_exporter.py (hunk 1 above)
Comment (akorosov) [reply nansat/tests/test_exporter.py]: That I can remove. that was rather for debugging of metadata.

File: nansat/tests/test_exporter.py (hunk 3 above)
Comment (akorosov) [reply nansat/tests/test_exporter.py]: That's needed only for passing the tests. I use GDAL directly to test the files independently (not with nansat). And it turns that it works differently on Win and Linux. On Win it read all metadata and doesn't add NC_GLOBAL, on Linux it shows only global metadata and adds 'NCGLOBAL' in front.

File: nansat/tests/test_exporter.py (hunk 2 above)
Comment (akorosov) [reply nansat/tests/test_exporter.py]: That I can also remove. I was experimenting why some files are not released after they are exported in windows. I though if I give some time it would help. Nope :(

**Comment by akorosov** [comment]
Well, since I already added a badge - it would be additional work to remove it and then to add again. I suggest to keep it.

**Comment by aleksandervines** [comment]
I don't get this. I fixed this NC_GLOBAL thing for windows in January. But it just disappeared from the code, and I can't trace where it happened.
https://github.com/nansencenter/nansat/commits/5197a5c15971e4d0d9186fa2a1262e329c813a58/nansat/tests/test_nansat.py

File: nansat/tests/test_exporter.py (hunk 3 above)
Comment (aleksandervines) [reply nansat/tests/test_exporter.py]: What about osx? I think you should default to nc_prefix = 'NC_GLOBAL#' unless it's windows - or possible better as I fixed this earlier, I checked if a metadata record existed with 'NC_GLOBAL#'

https://github.com/nansencenter/nansat/commit/172d31550bff4f5ef3ae7e10f893b13c86889fd3#diff-60a6bd1b9e79d00812bde1d1b98aba33

Comment (akorosov) [reply nansat/tests/test_exporter.py]: Good approach! I'm quite surprised it was not in the current branch...

Comment (akorosov) [reply nansat/tests/test_exporter.py]: Ahh, it probably disappeared when we were refactoring the code. Then we have split these tests from nansat_tests into exporter_tests.py

**Comment by akorosov** [comment]
It is not tested on OSX. Automatic testing on OSX (eg with TravisCI)  requires quite a bit of development/testing and belongs to a separate issue. 
It is safe to ignore testing at OSX at the moment since it is now OS independent and will be tested on OSX at a later stage (conda-forge) anyway.

**Comment by akorosov** [comment]
@korvinos , do you also approve the PR?
